tests/test_router.py

```diff
@@ -66,6 +66,17 @@ def test_router_hints_receives_filter_kwargs(self):
             list(TestModel.objects.filter(user_pk=self.user.pk))
             self.assertIn(lookups_to_find, read_route_function.call_args)
 
+    def test_router_hints_receives_get_kwargs(self):
+        test_model = TestModel.objects.create(user_pk=self.user.pk)
+
+        lookups_to_find = {'exact_lookups': {'user_pk': self.user.pk}}
+
+        with patch.object(ShardedRouter, 'db_for_read') as read_route_function:
+            read_route_function.return_value = 'app_shard_001'
```

Comment:
I changed this in my PR to use `self.user.shard` and it worked. Is there a reason it's hard coded as in the tests on my branch pass but with unexpected behavior?
https://github.com/JBKahn/django-sharding/pull/36